scripts/drmaa_external_runner.py

```diff
@@ -116,15 +121,20 @@ def set_user(uid, assign_all_groups):
 
 def main():
     userid, json_filename, assign_all_groups = validate_paramters()
-    set_user(userid, assign_all_groups)
+    # load JSON job template data before changing the user
```

Comment:
The switch in the order of the commands is not strictly necessary (I was just playing around -- but the problem that I was trying to solve was wrong permissions in the file system), but it will simplify my next PR. This will be to use `sudo -u ` instead of `sudo`